### PR TITLE
docs: clearer instructions in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by goobric**
changed the phrasing of the instructions to help make the understanding clearer. lines: 15, 21, 88 & 94
updated a typo line 13
- line 62, the . (dot) and *(asterisk) are not explicitly stated, and are easily lost in the meaning of this sentence. I mistakenly read, (not to run commands 'git add ' ) but then the following command is 'git add public'

## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1234"></a>

### CODE REVIEW

**Review by schmelto — COMMENTED** [review]
Here some thoughts from my side otherwise its looking good to me

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

File: CONTRIBUTING.md
Comment (goobric) [reply CONTRIBUTING.md:21]: hi Tom, thank you.
please add the colon :

File: CONTRIBUTING.md
Comment (goobric) [reply CONTRIBUTING.md:88]: hi Tom, thank you.
please add the colon :

File: CONTRIBUTING.md
Comment (goobric) [reply CONTRIBUTING.md:94]: hi Tom, thank you.
please add the colon :

**Comment by goobric** [comment]
@schmelto can you please explain to me:
- who is responsible for amending the CONTRIBUTING.md file?
- who is responsible for agreeing/disagreeing with the proposed changes?
- when am I informed if the proposed alterations to the file have been accepted or not?
- what does the label 'aspect: text' mean?

**Review by schmelto — APPROVED** [review]
## Looking good to me

Your suggested changes looking good to me.
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant

**Comment by schmelto** [comment]
> @schmelto can you please explain to me:
> 
> - who is responsible for amending the CONTRIBUTING.md file?

Not completely sure what you mean by this question 🤔


> - who is responsible for agreeing/disagreeing with the proposed changes?

The members of `@EddieHubCommunity/linkfree`


> - when am I informed if the proposed alterations to the file have been accepted or not?

You will see that others like me will approve the changes like it's done by me in the comment above. Currently two active maintainers habe to approve the pull request before the pull request is merged. 

> - what does the label 'aspect: text' mean?

It's only an internal flag that this pull request or issue contains changes that only affects texts of the repo so there is no coding change etc.

**Review by krshdev — APPROVED** [review]
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Comment by goobric** [comment]
April 16, 2022.
hi, @schmelto & @krshdev 
thank you for your comments regarding the issue raised.
Please excuse my naivety;
- what is the next stage in the process of updating the CONTRIBUTING.md document? (changes approved)
- when are the 'approved changes' evident/reflected within the current CONTRIBUTING.md document?
In relation to this question that I asked:
"who is responsible for amending the CONTRIBUTING.md file? Your response: Not completely sure what you mean by this question 🤔"
The members of the EddieHubCommunity/linkfree are granted the 'rights' to change the documentation; is this correct? Y/N
If 50% of the EHC-LF members agree with the changes and 50% do not agree with the changes proposed; Who has the overall responsibility as to whether these changes are implemented or not?
This is not clear to me! Sorry for being so 'dumm'

**Comment by krshdev** [comment]
> April 16, 2022.
> hi, @schmelto & @krshdev 
> thank you for your comments regarding the issue raised.
> Please excuse my naivety;
> - what is the next stage in the process of updating the CONTRIBUTING.md document? (changes approved)
> - when are the 'approved changes' evident/reflected within the current CONTRIBUTING.md document?
> In relation to this question that I asked:
> "who is responsible for amending the CONTRIBUTING.md file? Your response: Not completely sure what you mean by this question 🤔"
> The members of the EddieHubCommunity/linkfree are granted the 'rights' to change the documentation; is this correct? Y/N
> If 50% of the EHC-LF members agree with the changes and 50% do not agree with the changes proposed; Who has the overall responsibility as to whether these changes are implemented or not?
> This is not clear to me! Sorry for being so 'dumm'
> 

Hi, Any PR needs atleast 2 maintainer reviews before merging, there are two approvals, but I am not a maintainer 🙂 So you need to wait, till this PR gets more approvals.

The right to change the contributing guide is to everyone, but the amendment must be accepted by the maintainers/the leaders of the projects. The contributing guide is a guide for new members of the LinkFree Community, it is not the code of conduct of the project, which might have restricted editorial access.

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Review by kumarsonsoff3 — APPROVED** [review]
Thank you for your contribution! LGTM 🚀

**Comment by goobric** [comment]
I am still learning the tech terms and acronyms; can you please explain what 'LGTM' means? Thank you.
I am happy to contribute, and to help improve access for everyone.

**Comment by Pradumnasaraf** [comment]
> I am still learning the tech terms and acronyms; can you please explain what 'LGTM' means? Thank you. I am happy to contribute, and to help improve access for everyone.

Hey! goobric, sorry for the confusion, we try to make less use of the acronyms and jargon words.

**LGTM stands for - Looks good to me.**

**Comment by goobric** [comment]
220419 - Branch deleted

**Pull request merged**
